stats/release_queue.php

```diff
@@ -65,25 +65,29 @@
         echo "</tr>\n";
     }
 
-    $q_res = mysqli_query(DPDatabase::get_connection(), "
+    $q_sql = sprintf("
         SELECT *
         FROM queue_defns
-        WHERE round_id='$round_id'
-        ORDER BY ordering
-    ") or die(DPDatabase::log_error());
+        WHERE round_id='%s'
+        ORDER BY ordering",
+        DPDatabase::escape($round_id)
+    );
+    $q_res = DPDatabase::query($q_sql);
+
     while ( $qd = mysqli_fetch_object($q_res) )
     {
         $cooked_project_selector = cook_project_selector($qd->project_selector);
-        $c_res = mysqli_query(DPDatabase::get_connection(), "
+        $c_sql = sprintf("
             SELECT COUNT(*)
             FROM projects
             WHERE ($cooked_project_selector)
```

Comment:
`$cooked_project_selector` contains `%`s so we can't use `sprintf()` here (found by testing in my sandbox).

Reply:
Oof. OK, I've really got to check out how to use the sandboxes again.